posthog/temporal/data_imports/pipelines/pipeline.py

```diff
@@ -116,7 +118,10 @@ def _run(self) -> dict[str, int]:
                     )
                 except PipelineStepFailed as e:
                     # Remove once DLT support writing empty Delta files
-                    if isinstance(e.exception, DeltaError):
+                    if isinstance(e.exception, LoadClientJobRetry):
```

Comment:
Is the comment above related to this error?

Reply:
Yep - still relevant